.github/workflows/publish.yaml

```diff
@@ -0,0 +1,42 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        go-version: [1.11.x]
```

Comment:
This should probably match the other workflows with 1.13

Reply:
Yes please

Reply:
Updated